packages/block-library/src/cover/editor.scss

```diff
@@ -25,9 +33,7 @@
 }
 
 [data-align="left"] > .wp-block-cover,
-[data-align="right"] > .wp-block-cover,
-[data-align="left"] > .wp-block-cover-image,
-[data-align="right"] > .wp-block-cover-image {
```

Comment:
These are  for old versions and old versions are not loaded in the editor (they are converted)